src/main/java/core/basesyntax/bookstore/config/MapperConfig.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.bookstore.config;
+
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.NullValueCheckStrategy;
+
+@org.mapstruct.MapperConfig(

```

Comment:
You can import this annotation in the import statement 
```suggestion
@MapperConfig(
```